fontsize.md

```diff
@@ -0,0 +1,71 @@
+
+
+The problem
+-----------
+
+Convert an EBU-TT part 3 document to EBU-TT-D
+
+EBU-TT part 3 uses fonSize in 3 datatypes: cells, pixels and percentage.
+EBU-TT-D uses percentage fontSize only and prohibits pixels.
+
+In order to be converted the sizes need to be translated from pixels/cells to percentages.
+
+### Percentages vs. pixels/cells
+  The relationship between parent and child element is important when percentages are used as the child modulates the computed fontSize of the parent container, however it is an override when it comes to the other 2 types.
+  In order to be converted an absolute size needs to be calculated. According to TTML the [computed fontSize](http://w3c.github.io/ttml2/spec/ttml2.html#semantics-style-resolved-value-category-computed) value is an absolute value such as pixel. This poses a problem of tt@extent not always being available to us. cellResolution is available, as it has a default value and pixelAspectRatio is available as it has a default value.
+  
+### Percentages and the EM square
+  When the top level fontSize attribute is evaluated (say on the region) If it is a percentage value of 100% 100% the default fontsize EM square is scaled by maintaining its aspect ratio producing a square. ?? Is this true http://w3c.github.io/ttml2/spec/ttml2.html#style-attribute-fontSize seems to be contradictory on this matter ??
+  Then only one value of 100% is provided according to TTML I am supposed to assume 100% 100% that gives the same square result.
+  
+  Let's look at 1c 1c. Since we have a cellResolution of 32:15 this should mean that the EM square is anamorphically scaled to 1c 1c assuming that the pixelAspectRatio is 1:1 and extent width:extent height != 32:15 (The root container region is not a square)
```

Comment:
would be good to action these two comments in fontsize.md just so that people later do not come along and get the wrong idea.